pkg/runner/job_runner.go

```diff
@@ -228,7 +245,7 @@ func (jr *JobRunner) Run(ctx xcontext.Context, j *job.Job, resumeState *job.Paus
 					close(stopRefresh)
 					<-stoppedRefresh
 					// Refresh one last time.
-					if err2 := tl.RefreshLocks(runCtx, j.ID, targets); err2 != nil {
+					if err2 := tl.Lock(runCtx, j.ID, jr.targetLockDuration, targets); err2 != nil {
```

Comment:
oh, good catch. this is a leftover from early version of the change where i actually thought we didn't need RefreshLocks and could do with Lock everywhere - since before this change Refresh was just Lock with a different (and constant) timeout. when i went to parametrize the timeout used by refresh i realized that it was exactly the same as Lock and wanted to get rid of it. but then, after more thought i realized that no, we needed a separate operation with different semantics.